### DISCUSSION TITLE
No more weather data import as of today (22-May-2023) ?

### DISCUSSION THREAD

**Discussion by froggyranita2** [mytourbook/mytourbook#1113] [Q&A] [answered]
Hello, when i try to import weather data i receive that error message:

"Erreur lors de la récupération de données météorologiques: "{"cod":429,"message":"Your account is temporary blocked due to exceeding of requests limitation of your subscription type. Please choose the proper subscription https://openweathermap.org/price"}"

Regards, Pascal

**Comment by FJBDev** [comment]
Pascal,

It seems that the OpenWeatherMap provider has been overused recently. 
Please try `WeatherApi` in the meantime

Frederic

**Reply by FJBDev** [reply]
I will update here when `OpenWeatherMap` unblocks the account

**Reply by witdem** [reply]
Weatherapi is working 

Regards, Maarten

**Comment by FJBDev** [comment, marked as answer]
The weather retrieval with `OpenWeatherMap` is now working again.
Let me know if you still have issues

![image](https://github.com/mytourbook/mytourbook/assets/25235665/b58ea11a-e703-4d46-bacc-382d4aaa651a)

**Reply by froggyranita2** [reply]
Hello Frederic, i confirm the import of weather data using OpenWeatherMap is working again.
Merci. Pascal
